.github/workflows/ci.yml

```
@@ -46,6 +46,35 @@ jobs:
     - name: Spell Check
       uses: crate-ci/typos@master
 
+  # Run 32-bit support check separately.
+  bit-32-support:
```

Maybe makes more sense to specify WASM 32-bit instead of general 32-bit which we probably won't/can't support.
```suggestion
  # Run 32-bit WASM support check separately.
  wasm-32-bit-support:
```